InnerEye/ML/Histopathology/models/deepmil.py

```diff
@@ -307,6 +322,20 @@ def test_epoch_end(self, outputs: List[Dict[str, Any]]) -> None:  # type: ignore
                 fig = plot_slide_noxy(slide, score, paths, bottom_attn, key + '_bottom', ncols=4)
                 figpath = Path(key_folder_path, f'{slide}_bottom.png')
                 fig.savefig(figpath, bbox_inches='tight')
+                if len(self.slide_dataset) > 0:
+                    slide_dict = list(filter(lambda entry: entry[SlideKey.SLIDE_ID] == slide, self.slide_dataset))[0]  # type: ignore
+                    load_image_dict(slide_dict, level=self.level, margin=0)                                            # type: ignore
+                    slide_image = slide_dict[SlideKey.IMAGE]
+                    location_bbox = slide_dict['location']
```

Comment:
Can we replace the literal "location" with a SlideKey enum?

Reply:
I changed this, thanks.